packages/aws-cdk-lib/aws-appsync/test/appsync-rds.test.ts

```diff
@@ -112,9 +127,88 @@ describe('Rds Data Source configuration', () => {
     });
   });
 
+  test('appsync creates correct policy ServerlessV2', () => {
```

Comment:
To keep this more organized, we could separate v1 and v2 tests in dedicated `describe` blocks.

Reply:
@lpizzinidev, sorry missed it. updated.